### ISSUE TITLE
Redesigned Workflow & Dashboard - progress indicators

### ISSUE THREAD

**Issue by jardakotesovec** [pkp/pkp-lib#10663]
Description to be added, just to document intentions and having good starting point for further evolution in next version(s).

**Comment by Devika008** [comment]
Hi Jarda,

I hope you can bear with me as I throw a few changes your way—I promise I’ve thought this through! 😄

### Dashboard (submission listing)
Indicates whenever listing is being re-fetched for any reason

We shouldn’t place loaders next to "Filters" as it implies that a change or reset has been made to the filters, which could be misleading. Additionally, the placement is too small to notice that something is being loaded. To ensure clarity and consistency, I had proposed placing the loaders at the top.
To reduce user anxiety, we should also accompany the loaders with text, as loaders can sometimes appear endless in certain cases or installations.

Link: https://youtu.be/jOBjaAzL2kY

### Workflow Page
indicates when any of the components inside workflow page are fetching/refetching data

Rule of thumb is that loaders need to always be placed on the top unless a specific section like just the document tables is being changed. We shouldnt be placing there as it breaks the UX logic of loaders in general. Again to reduce user anxiety, we should also accompany the loaders with text, as loaders can sometimes appear endless in certain cases or installations.

Link: https://youtu.be/mr4rlHvbZVE

The full-page loaders work well, but I strongly recommend accompanying them with text. Leaving them as endless loaders isn’t a best practice and can create unnecessary confusion or frustration for users.

**Comment by Tribunal33** [comment]
@jardakotesovec looks like this was assigned to me but is it ready for testing?  I couldn't trigger the progress indicator commments indicated on the figma designs.  I checked the code for the comments as throttling and setting breakpoints didn't reveal these progress indicator comments at the first 15, 15-30, and 30+ second mark. : 
"Loading please wait..."
"Still loading, but almost there..."
"Thanks for your patience, we're still processing your data..."

Let me know if there are better ways to test these progress indicators.  The code is too efficient to trigger them :).

**Comment by jardakotesovec** [comment]
@Tribunal33 These are from earlier designs. Its simplified now, so we don't use these texts. I think its updated on figma as well.

**Comment by Tribunal33** [comment]
@jardakotesovec I don't know if this is in scope but I noticed that the exporting of users progress indicator never ends.

There really aren't many steps to reproduce.
Go to Tools -> Users XML Plugin -> Export Users -> Select some users -> Export users

File downloads to broswser but the progress indicator never stops spinning.  I found it when testing the security issue with importing admin user role_ids.

If it is out of scope I could write up a bug for it to be fixed later too.  

![Image](https://github.com/user-attachments/assets/3ad719c6-f024-4ee7-8b1e-cbd37c84d10e)

**Comment by jardakotesovec** [comment]
@Tribunal33 Yes that would be out of scope..  what I have done on workflow/dashboard should not interfere with progress bars elsewhere.

**Comment by Tribunal33** [comment]
@jardakotesovec thanks.  I created a bug for it #10801 .  This ticket passes QA.  

Test Case Overview
1.	Test each of the states by copying to incognito browser tab
a.	The view
b.	The filters
c.	Search phrases
d.	Column sorting
e.	Opened submission id
f.	Selected side menu in submission workflow
2.	Reverse the param states beginning with full params in the URL
a.	Close the submission
b.	Clear filters
c.	Cancel search
3.	Test user access to these URLs
a.	Author shouldn’t have Access to wrong submission ids
b.	Readers shouldn’t have access to certain views
